airgun/views/discoveryrule.py

```diff
@@ -0,0 +1,76 @@
+from widgetastic.widget import (
+    Checkbox,
+    Text,
+    TextInput,
+    View,
+)
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    FilteredDropdown,
+    MultiSelect,
+    SatTable,
+)
+
+
+class DiscoveryRulesView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Discovery Rules']")
+    new = Text("//a[contains(@href, '/discovery_rules/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class DiscoveryRuleCreateView(BaseLoggedInView):
+    name = TextInput(locator="//input[@id='discovery_rule_name']")
```

Comment:
These inputs are inside 'Primary' tab. And btw 'Locations' and 'Organizations' tabs are missing for DiscoveryRuleCreateView